Test/MainWindow.xaml.cs

```diff
@@ -75,7 +75,7 @@ private void ButtonClick(object sender, RoutedEventArgs e)
 		protected virtual void OnPropertyChanged([CallerMemberName] string propertyName = null)
 		{
 			PropertyChangedEventHandler handler = PropertyChanged;
-		    handler?.Invoke(this, new PropertyChangedEventArgs(propertyName));
+		    handler.Invoke(this, new PropertyChangedEventArgs(propertyName));
```

Comment:
This is a conditional operator since handler could be null. Why did you remove this? 
http://www.kunal-chowdhury.com/2014/12/csharp-6-null-conditional-operators.html?m=1